_posts/2022/2022-01-27-recruitment-drive-2022.md

```diff
@@ -0,0 +1,71 @@
+
+Although everyone can participate in our recruitment drive, the majority of our recruits are in their 1st year of study.
+
+# How do you get in?
+
+There are many ways to land an interview at SDSLabs during the Recruitment Drive. These methods are:
+
+* Being amongst the top teams at the SDSLabs' Syntax Error Hackathon 2022
+* Completing SDSLabs’ Winter of Code 2022
+* Being amongst the top students at any of our competitions mentioned below
+* Being amongst the top students in the SDS Recruitment Test and Design problem statement submissions
+
+## Syntax Error
+
+Syntax Error is a campus-level hackathon conducted by us where students come up in teams and work together to build some amazing applications from scratch.
+
+We call first-year students who are placed in the top 3 teams in the overall category for a direct interview.
+
+## Winter Of Code 2022
+
+Students who were selected for SDSLabs’ Winter of Code 2022 and were able to pass all their evaluations successfully will be invited for an interview at SDSLabs during the interview process.
+
+
+## Competitions
+
+SDS organizes a variety of competitions for the 1st year junta at IIT Roorkee. We like to call the Top 3 students from each of these competitions for an interview:
+
+* NoobCTF
+  * Capture the flag competition
+  * Hosted at [Backdoor](https://backdoor.sdslabs.co/) with InfoSec IITR
+* Beginner's Hypothesis
+  * Data Science competition
+  * Hosted at [Cerebro](https://cerebro.sdslabs.co/) with DSG IITR
+* Algophobic
+  * Competitive programming competition
+  * Hosted at [Codeforces](https://codeforces.com/) with PAG IITR
+* Endgame
+  * Mathematical problem-solving competition
+  * Hosted at [Erdős](https://erdos.sdslabs.co/)
+
+Freshers with an exceptional submission at the [Global Game Jam 2022](https://globalgamejam.org/2022/jam-sites/sdslabs) might also get a direct interview call during the recruitment drive.
+
+## Recruitment Test and Design Problem statements
+
+SDSLabs conducts a recruitment test for freshers, having various sections including logical reasoning, programming, software development, and an ad-hoc section to show off your geeky side.
+
+We invite the top scores from individual sections and the top scorers overall for an interview during the interview process. Needless to say, the evaluation will be done based on your performance in individual sections.
+
+Design based problem statements are released a week before the recruitment test takes place to test your skills in various fields of design like UI/UX and graphic design. Design submissions will be considered for getting recruited as a Designer at SDSLabs.  
+
+<br />
+
+Apart from this, we are yearly open to wildcard entries, open to new members irrespective of year and branch. Obsessed with coding? Building something new every week? We insist that you join us! Send us your resume, at: [contact@sdslabs.co.in](mailto:contact@sdslabs.co.in).
```

Comment:
~yearly~ always
open to new members who may have missed their shot in the recruitment drive